tests/roles/mariadb_copy/templates/dump_dbs.bash

```diff
@@ -6,7 +6,52 @@
 mkdir -p {{ mariadb_copy_tmp_dir }}
 cd {{ mariadb_copy_tmp_dir }}
 
-podman run -i --rm --userns=keep-id -u $UID -v $PWD:$PWD:z,rw -w $PWD $MARIADB_IMAGE bash <<EOF
+oc apply -f - <<EOF
+---
+apiVersion: v1
+kind: PersistentVolumeClaim
+metadata:
+  name: mariadb-data
+spec:
+  storageClassName: $STORAGE_CLASS_NAME
```

Comment:
Looking at the CI failure, this needs to go among `mariadb_copy_shell_vars_src`?